src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -165,6 +174,14 @@ def _process_message(wrapper: Dict) -> None:
                 if duration is None:
                     duration = int((start_time - check_in.date_added).total_seconds() * 1000)
 
+                if not valid_duration(duration):
+                    metrics.incr(
+                        "monitors.checkin.result",
+                        tags={**metric_kwargs, "status": "failed_duration_check"},
+                    )
+                    logger.debug("check-in duration is invalid: %s", project.organization_id)
```

Comment:
not sure if i wanted to move this logic to the helper method, but then i'd have the pass the params which I don't really want to do. long term we should have this logic run through the validator